satpy/etc/enhancements/generic.yaml

```diff
@@ -405,3 +405,108 @@ enhancements:
         stretch: crude
         min_stretch: [0]
         max_stretch: [0.075]
+  snow_age_default:
+    standard_name: snow_age
+    operations:
+    - name: snow_age
+      method: !!python/name:satpy.enhancements.lookup
+      kwargs:
+       luts: [[0, 0, 0], [1, 2, 2], [3, 8, 5], [4, 12, 8], [6, 15, 10], [8, 18, 13], [9, 21, 16],
```

Comment:
Is it possible to have the LUT in a separate file, like .npy, .npz or ascii?

Reply:
Disregard the previous comment on the LUT, in this case it's practically static data and is OK to have it in the YAML. The `lookup()` function is ignorant about the source anyway, so can be used for other things, too.